src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,60 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { useState } from 'react';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard/MovieCard';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+type Props = {
+  query: string;
+  setQuery: (query: string) => void;
+  addMovie: (movie: Movie) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({ query, setQuery, addMovie }) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [isError, setIsError] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
+
+  const handleInput = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setQuery(event.target.value);
+    setIsError(false);
+  };
+
+  const clearMovie = () => {
+    setMovie(null);
+  };
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    try {
+      setIsLoading(true);
+      const foundMovie = await getMovie(query);
+
+      if ('Error' in foundMovie) {
+        setIsError(true);
+      } else {
```

Comment:
Better to do it in api file, where you making a request, and throw an error if there is error in responce

Then you'll be able to error here and show it on UI